README.md

```diff
@@ -59,6 +59,32 @@ $ openssl genrsa 2048 > ../localhost.key
 $ openssl req -new -x509 -nodes -sha256 -days 3650 -key ../localhost.key -subj '/CN=*.localhost' > ../localhost.cert
 ```
 
+### Single-user server with Docker
+
+(First, install [Docker](https://docker.com)).
```

Comment:
I would strongly consider linking to the installation documentation specifically:

https://docs.docker.com/engine/installation/

This avoids people following the "get started" link on the homepage which will take you to different documentation that is significantly worse.